assets/styles/viper.css

```diff
@@ -28,25 +30,65 @@ circle {
 .footer {
   position: absolute;
   bottom: 0;
-  height: 22%;
+  height: 25%;
```

Comment:
Padding fixed. Code windows scale keeping 430px as minimum (to correctly display buttons). Please merge.